tests/testthat/dummy_packages/assignmentLinter/DESCRIPTION

```diff
@@ -0,0 +1,9 @@
+Package: assignmentLinter
```

Comment:
To keep the `tests/testthat` folder neat, could the other dummy package (`tests/testthat/package`) be moved into `tests/testthat/dummy_packages/`. The tests that use `package` are all in `test-dir_linters.R` and they will require their filepaths to be redirected.